### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
@vikivivi please review

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4975,6 +4976,9 @@ canbe->can be
 cancelaltion->cancellation
 cancelation->cancellation
 cancelations->cancellations
+cancele->cancel
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:4979]: Suggest `cancels` as well. Key `e` and `s` is nearby
```text
cancele->cancel, cancels,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -14072,6 +14077,7 @@ expierience->experience
 expieriences->experiences
 expilicitely->explicitly
 expireitme->expiretime
+expiried->expired
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:14080]: If you do not mind add these
```text
expirie->expire
expiriation->expiration
expiriy->expiry
```
* https://github.com/search?q=expiriation&type=code
* https://github.com/search?q=expirie&type=code
* https://github.com/search?q=expiriy&type=code

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -27034,6 +27041,7 @@ resoruce->resource
 resoruced->resourced
 resoruces->resources
 resorucing->resourcing
+resotred->restored, resorted,
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:27044]:
```text
resotration->restoration
resotrations->restorations
resotrative->restorative
resotre->restore
resotrer->restorer
resotrers->restorers
resotres->restores
resotring->restoring
```
* https://github.com/search?q=resotration&type=code
* https://github.com/search?q=resotrations&type=code
* https://github.com/search?q=resotrative&type=code
* https://github.com/search?q=resotre&type=code
* https://github.com/search?q=resotred&type=code
* https://github.com/search?q=resotrer&type=code
* https://github.com/search?q=resotrers&type=code
* https://github.com/search?q=resotres&type=code
* https://github.com/search?q=resotring&type=code

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -27612,6 +27620,7 @@ sacrifical->sacrificial
 sacrifying->sacrificing
 sacrilegeous->sacrilegious
 sacrin->saccharin
+saddly->sadly
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:27623]: Suggest `saddle` as well.
```text
saddly->sadly, saddle,
```

**Review by vikivivi — CHANGES_REQUESTED** [review]

**Comment by luzpaz** [comment]
@vikivivi Added in [109b183](https://github.com/codespell-project/codespell/pull/2386/commits/109b183ebc4e0f8672b8f45b5a4cc13a7aa2a455)

**Pull request merged**